pyproject.toml

```diff
@@ -104,4 +105,14 @@ fail-under = 5.6
 
 [tool.pylint.format]
 max-line-length = 88
-max-module-lines = 800
\ No newline at end of file
+max-module-lines = 800
+
+[tool.mypy]
+files = [
+    'noxfile.py',
+    'scripts/**/*.py',
+    # Incrementaly add ad fix files and paths until entire project is checked
+]
+mypy_path = [
+    'typeshed'
```

Comment:
what does this?

Reply:
It adds the directory to the search path where mypy is looking for type specifications. pyodbc has some specs but sadly they do not install them correctly so mypy can't pick it up from the installed package. Until that get's fixed I thought having the pydobc typedefs in the project is the best solution for now.